src/ompl/base/PlannerTerminationCondition.h

```diff
@@ -120,6 +121,32 @@ namespace ompl
 
         /** \brief Return a termination condition that will become true \e duration seconds in the future (wall-time), but is checked in a separate thread, every \e interval seconds; \e interval must be less than \e duration */
         PlannerTerminationCondition timedPlannerTerminationCondition(double duration, double interval);
+
+        /** \brief Return a termination condition that will become true as soon as the problem definition has an exact solution */
+        PlannerTerminationCondition exactSolnPlannerTerminationCondition(ompl::base::ProblemDefinitionPtr pdef);
```

Comment:
Oh yeah, I never thought of that.

There may be some use in conjunction with an and/or PTC (i.e., setting a *minimum* run time by ANDing with a timedPTC), but I can't vouch for whether there's any actual need for that.

As I said, I'm not overly attached to any of these, so it's whatever you want. :smile: